### PR TITLE
Fixed phone number related problem

### PR DESCRIPTION
**Pull request by YuZizhen**
Fix the wrong phone number in example.

Change description for phone number.

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -105,14 +105,15 @@ Below is a sample unsuccessful command executed. Note that the command is highli
   * All input values are case-sensitive unless otherwise specified.
 
 * Duplicate contacts:
-  * Contacts are considered duplicates if they share the same phone number.
+  * Contacts are considered duplicates if they share the same mobile phone number.
+  * We consider mobile phone number as an unique identifier of a person in Singapore because in most of the cases, two people will not share the same mobile number (similar to how PayLah and PayNow works)
```
Comment (KohGuanZeh) [inline docs/UserGuide.md:109]:
```suggestion
  * Contacts are considered duplicates if they share the same mobile phone number and is used as unique identifier of a Person as no two persons would share the same mobile phone number.
```
Would this be better?

**Review by KohGuanZeh — APPROVED** [review]

File: docs/UserGuide.md [hunk 2]
```diff
@@ -105,14 +105,14 @@ Below is a sample unsuccessful command executed. Note that the command is highli
   * All input values are case-sensitive unless otherwise specified.
 
 * Duplicate contacts:
-  * Contacts are considered duplicates if they share the same phone number.
+  * Contacts are considered duplicates if they share the same mobile phone number and is used as unique identifier of a Person as no two persons would share the same mobile phone number.
```
Comment (Javiery3889) [inline docs/UserGuide.md:108]:
```suggestion
* In FINDvisor, contacts are **uniquely identified** using their mobile phone number. Hence, two contacts are considered duplicates if they share the same mobile phone number. 
```
@KohGuanZeh @YuZizhen Just a nit, but does this sound better?

Comment (KohGuanZeh) [reply docs/UserGuide.md:108]: Looks good to me. I think we should adjust it to this.

File: docs/UserGuide.md [hunk 3]
```diff
@@ -105,14 +105,14 @@ Below is a sample unsuccessful command executed. Note that the command is highli
   * All input values are case-sensitive unless otherwise specified.
 
 * Duplicate contacts:
-  * Contacts are considered duplicates if they share the same phone number.
+* In FINDvisor, contacts are **uniquely identified** using their mobile phone number. Hence, two contacts are considered duplicates if they share the same mobile phone number.
```
Comment (Dethada) [inline docs/UserGuide.md:108]: I think you are missing an indentation here, also remove the trailing space.

**Review by KohGuanZeh — COMMENTED** [review]
Apart from the fix @Dethada mentioned, LGTM

**Review by KohGuanZeh — APPROVED** [review]

**Review by Javiery3889 — APPROVED** [review]

**Pull request merged**